### PR TITLE
SED-1079 notificationSubscribtionSelectionTable migration

### PR DESCRIPTION
**Pull request by bojidaryovchev**
- look aligned with the old version (padding)
- tooltips and such not visible inside old bootstrap modals (z-index)

### CODE REVIEW

File: projects/step-core/src/lib/modules/table/components/table/table.component.scss [hunk 1]
```diff
@@ -15,8 +15,9 @@
     width: auto !important;
   }
 
-  ::ng-deep tr > td.mat-cell:not(:first-of-type):not(.mat-column-actions) {
-    padding: 0 8px;
+  ::ng-deep tr > td.mat-cell:not(:first-of-type):not(.mat-column-actions),
+  ::ng-deep tr > th.mat-header-cell:not(:first-of-type):not(.mat-column-actions) {
+    padding: 8px;
```
Comment (dvladir) [inline projects/step-core/src/lib/modules/table/components/table/table.component.scss:20]: Is this change required?
As far as I remember original line that was removed, was the part of the design fixes

**Review by neogucky — APPROVED** [review]
Looks fine, still we will review the padding: 8px (so top bottom are new) after merging.
